conda/win/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set data = load_setup_py_data(setup_file='conda/_conda_vers.py') %}
+
+package:
+  name: psi4
+  version: {{ data.get('version') }}
+
+source:
+  path: ../..
+
+requirements:
+  run:
+    - deepdiff
+    - intel-openmp={{ MKL_VERSION }}
+    - mkl={{ MKL_VERSION }}
+    - networkx
+    - numpy
+    - pytest
+    - python={{ PY_VER }}
```

Comment:
Ultimately, I think we'll want to break out separate packages for deps and to use `run_exports` and `pin_compatible` for version constraints, but great for now.

Reply:
Yes, I agree! Currently, `libxc` and `libinit` are statically linked. So the dependency shouldn't be a problem.

Reply:
Completely agree -- can't cause trouble as-is.